docs/asus-rog-ryujin-guide.md

```diff
@@ -0,0 +1,91 @@
+# ASUS ROG RYUJIN II liquid cooler
+_Driver API and source code available in [`liquidctl.driver.asus_rog_ryujin`](../liquidctl/driver/asus_rog_ryujin.py)._
+
```

Comment:
Missing `_New in git._<br>`. note here ([guide](https://github.com/liquidctl/liquidctl/blob/main/docs/developer/process.md#newchanged-in-notes)).